tinyagentos/routes/agent_import.py

```
@@ -0,0 +1,369 @@
+            if agent is not None:
+                agent["status"] = "failed"
+            err_msg = str(exc)
+            deploy_tasks[unique_slug] = {
+                "status": "failed", "name": unique_slug, "error": err_msg,
+            }
+            await _notify_failure(request, unique_slug, err_msg)
+        finally:
+            _safe_unlink(_BUNDLE_PATHS.pop(unique_slug, None))
+            await save_config_locked(config, config.config_path)
+
+    asyncio.create_task(_background_import())
```

Ruff (0.15.18)

[warning] Store a reference to the return value of `asyncio.create_task`

(RUF006)